src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -430,50 +431,23 @@ string relationshipEntity
         }
 
         /// <summary>
-        /// Test method to check that an exception is thrown when relationship exists between the wrong pair. i.e.,
-        /// LinkingSourceField OR SourceFields are null and relationship between target and linking object exists, no relationship between source and linking object
-        /// LinkingTargetField OR TargetField are null but relationship between source and linking object exists, no relationship between target and linking object
-        /// Also Test when the foreignKeyPair exist for correct pair, no exception is thrown. i.e.,
-        /// LinkingSourceField OR SourceFields are null but relationship between source and linking object exists, no relationship between target and linking object
-        /// LinkingTargetField OR TargetField are null but relationship between target and linking object exists, no relationship between source and linking object
+        /// Test method to check that an exception is thrown when LinkingObject is null
```

Comment:
It would be clearer to explicitly call out that this test validates config written for 1:many, many:1, 1:1 and NOT many:many. Many:Many is excluded from this test because we intentionally don't define a linking object/linking fields. 

You're essentially testing that this config is valid when a fk def exists in the database relating series and book. and that this config results in startup failure when a fk def doesn't exist in the database relating series and book.

```json
"Series": {
  "source": "dbo.series",
  "relationships": {
    "books": {
      "cardinality": "many",
      "target.entity": "Book",
      "source.fields": null,
      "target.fields": null
    }    
  }
}
```

Reply:
in other words, it's a little confusing when the first condition that this comment describes is the lacking of **linking** object.